conda_forge_tick/migrators.xsh

```diff
@@ -427,7 +427,14 @@ class Noarch(Migrator):
 
     def migrate(self, recipe_dir, attrs, **kwargs):
         with indir(recipe_dir):
-            replace_in_file(
+            if '    script:' in attrs['raw_meta_yaml']:
```

Comment:
This might not work. You should get the number of spaces in the line after `build:`.